app/views/models/welcome.html.erb

```diff
@@ -1,11 +0,0 @@
-<% content_for(:title, _("Hardware Models")) %>
```

Comment:
FYI, this will need to undo the changes at https://github.com/theforeman/foreman/pull/6818/ as well.